src/ape/utils.py

```diff
@@ -444,6 +444,16 @@ def get_all_files_in_directory(path: Path) -> List[Path]:
     return [path]
 
 
+class dependency(property):
```

Comment:
I feel like `dependency` is starting to become an overloaded term.

Users may think this has something to do with project dependencies, like Open Zeppelin?

Maybe a better name for this is `placeholder(property)` or something?

Reply:
`injected_property` is good!

Reply:
or how about `to_be_injected` or `injected_before_use`? To show how to use it